walletnode/api/design/artworks.go

```diff
@@ -119,6 +119,45 @@ var _ = Service("artworks", func() {
 			Response(StatusCreated)
 		})
 	})
+
+	Method("artSearch", func() {
+		Description("Streams the search result for artwork")
+		Meta("swagger:summary", "Streams the search result for Artwork")
+
+		Payload(SearchArtworkParams)
+		StreamingResult(func() {
+			Extend(ArtworkTicket)
+			Attribute("image", Bytes, func() {
+				Description("Thumbnail image")
+			})
```

Comment:
Why not move `image` into `ArtworkTicket` type?

Reply:
@levkoburburas I'm using an already existing object in this file, I'm reusing that & with this extend ...the image essentially becomes a part of the object for this API